src/lib/fhevm/fhevmjs.ts

```diff
@@ -0,0 +1,141 @@
+import { isAddress } from "ethers";
+import { initFhevm, createInstance, FhevmInstance } from "fhevmjs/bundle";
+
+const ACL_ADDRESS: string | undefined = process.env.NEXT_PUBLIC_ACL_ADDRESS;
+const KMSVERIFIER_ADDRESS: string | undefined =
+  process.env.NEXT_PUBLIC_KMS_ADDRESS;
+const GATEWAY_URL: string | undefined = process.env.NEXT_PUBLIC_GATEWAY_URL;
```

Comment:
should these be loaded from `env` like in `src/config/index.tsx`
```suggestion
import {
  NEXT_PUBLIC_ACL_ADDRESS as ACL_ADDRESS,
  NEXT_PUBLIC_KMS_ADDRESS as KMSVERIFIER_ADDRESS,
  NEXT_PUBLIC_GATEWAY_URL as GATEWAY_URL,
} from "./../config/env";
```